final_task/rss_reader/rss_reader.py

```diff
@@ -0,0 +1,28 @@
+import argparse
+from Handler import Handler
+from Logging import logging_decorator
+from Console_parse import get_arguments_from_console
```

Comment:
I do not see these files in this PR. Are they 3rd-party libraries? If so, please update requirements.txt file

Reply:
I forgot to commit these files.

Reply:
Still do not see these files. There is nothing for me to review